ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/Entry/Plugin.cs

```diff
@@ -184,7 +184,7 @@ protected override LoadReturnCode OnLoad(ref string errorMessage)
 
       var hostAppName = HostApplications.Rhino.Slug;
       var hostAppVersion = Utils.GetRhinoHostAppVersion();
-      SpeckleLog.Initialize(HostApplications.Rhino.Slug, Utils.GetRhinoHostAppVersion());
+      SpeckleLog.Initialize(HostApplications.Rhino.Slug, Utils.GetRhinoHostAppVersion(), logConfig);
```

Comment:
wait, does this completely fix it?

Reply:
Answered on Discord but copying it over here:

we obviously had this serilog.exceptions issue in the past, so we added a property enhancedLogContext to the loggerConfiguration that we could flip to have that non initialised.

This "fix" was reverted accidentaly when we fixed the issue with logs being mixed between Rhino/Grasshopper, and the logConfig was no longer being passed down

This caused the config to be the default, which sets enhancedLogContext as true, and runs that chunk of code that calls EnrichWithExceptions()